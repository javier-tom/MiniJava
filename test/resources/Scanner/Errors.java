int five = 5 /* 4;

_eleven