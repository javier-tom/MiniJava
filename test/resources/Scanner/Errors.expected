INT ID(five) BECOMES INTEGER_LITERAL(5) <UNEXPECTED(/)> TIMES INTEGER_LITERAL(4) SEMICOLON <UNEXPECTED(_)> ID(eleven)